R/Figures.R

```diff
@@ -864,30 +864,62 @@ TADA_TwoCharacteristicScatterplot <- function(.data, id_cols = "TADA.ComparableD
   }
 
   if (!"TADA.ComparableDataIdentifier" %in% id_cols) {
-    print("Note: TADA.ComparableDataIdentifier not found in id_cols argument and is highly recommended.")
+    stop("Note: TADA.ComparableDataIdentifier not found in id_cols argument.")
   }
 
   # check that groups are in id_cols
   id <- unlist(unique(.data[, id_cols]))
   if (any(!groups %in% id)) {
     stop("The 'groups' vector contains one or more inputs that are not found within your input dataset. Check spelling and try again.")
   }
+  
+  # check that groups are length two for comparing two characteristics
+  if (length(groups) != 2) {
+    stop("The 'groups' vector does not contain two characteristics to compare.")
```

Comment:
Should this first "groups" param be able to accept only one characteristic? For cases when you want to compare one characteristic at multiple locations?